bin/prepare-worker/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+/// Prepare worker binary name
+pub const BINARY_NAME: &str = "polkadot-prepare-worker";
```

Comment:
It's the same as `PREPARE_BINARY_NAME` in pvf-host. Can we move it to the pvf/prepare-worker crate, and use that here, for less duplication?